superset/db_engine_specs/base.py

```diff
@@ -959,3 +963,20 @@ def mutate_db_for_connection_test(database: "Database") -> None:
         :param database: instance to be mutated
         """
         return None
+
+    @staticmethod
+    def get_extra_params(database: "Database") -> Dict[str, Any]:
+        """
+        Some databases require adding elements to connection parameters,
+        like passing certificates to `extra`. This can be done here.
+
+        :param database: database instance from which to extract extras
+        """
```

Comment:
nit: it's nice to add a `:raises:` on the docstring when a method can raise an exception

Reply:
Ah, missed that one, will update.